plugins/modules/src/migrate/migrate.go

```diff
@@ -468,6 +475,22 @@ func main() {
 		ansible.FailJson(response)
 	}
 
+	// Handle Volume type mapping
```

Comment:
Overall looks good to me.
I would probably move that logic into a utils package or specific function, but it's just comestic.
I have tested the mapping logic and it's working good.

Thank you for PR.